ui/raidboss/data/03-hw/raid/a12s.js

```diff
@@ -5,4 +5,246 @@
+        'Sacrament': 'Sakrament',
+        'Smash': 'Schmettern',
+        'Summon Alexander': 'Alexanders Beschwörung',
+        'Tank Buster': 'Tank Buster', // FIXME
+        'Temporal Stasis': 'Zeitstillstand',
+        'Tethers': 'Tethers', // FIXME
+        'Tetrashatter': 'Kristallbruch',
+        'Timegate': 'Zeittor',
+        'Void of Repentance': 'Kammer der Buße',
+        'Wings Spawn': 'Wings Spawn', // FIXME
+      },
+    },
+    {
+      'locale': 'fr',
+      'replaceSync': {
+        'Removing combatant Alexander Prime': 'Removing combatant Alexander Prime', // FIXME
+        'casts Holy Bleed': 'casts Holy Bleed', // FIXME
+        'casts Void of Repentance': 'casts Void of Repentance', // FIXME
+        'readies Chronofoil': 'readies Chronofoil', // FIXME
+        'uses Almost Holy': 'uses Almost Holy', // FIXME
+        'uses Divine Spear': 'uses Divine Spear', // FIXME
+        'uses Smash': 'uses Smash', // FIXME
+      },
+      'replaceText': {
+        '--Boss Targetable--': '--Boss Targetable--', // FIXME
+        '--Boss Untargetable--': '--Boss Untargetable--', // FIXME
+        '--Reset--': '--Réinitialisation--',
+        'Almost Holy': 'Quasi-Miracle',
+        'Arrhidaeus Spawn': 'Arrhidaeus Spawn', // FIXME
+        'Blazing Scourge': 'Lumière fustigeante',
+        'Chastening Heat': 'Chaleur de l\'ordalie',
+        'Chronofoil': 'Ailes du temps',
+        'Cleave': 'Cleave', // FIXME
+        'Cross Beams': 'Cross Beams', // FIXME
+        'Crystal Explode': 'Crystal Explode', // FIXME
+        'Divine Judgment': 'Jugement divin',
+        'Enrage': 'Enrage',
+        'Gravitational Anomaly': 'Anomalie gravitationnelle',
+        'Half Gravity': 'Demi-Pesanteur',
+        'Holy Bleed': 'Impact miraculeux',
+        'Holy Scourge': 'Lumière fustigeante',
+        'Inception': 'Commencement',
+        'Incinerating Heat': 'Chaleur purifiante',
+        'Judgment Crystal': 'Cristal du jugement',
+        'Lasers': 'Lasers', // FIXME
+        'Mega Holy': 'Méga Miracle',
+        'Might & Time Spawn': 'Might & Time Spawn', // FIXME
+        'Radiant Sacrament': 'Sacrement rayonnant',
+        'Sacrament': 'Sacrement',
+        'Smash': 'Fracassement',
+        'Summon Alexander': 'Invocation d\'Alexander',
+        'Tank Buster': 'Tank Buster', // FIXME
+        'Temporal Stasis': 'Stase temporelle',
+        'Tethers': 'Tethers', // FIXME
+        'Tetrashatter': 'Rupture',
+        'Timegate': 'Porte temporelle',
+        'Void of Repentance': 'Vide du repentir',
+        'Wings Spawn': 'Wings Spawn', // FIXME
+      },
+    },
+    {
+      'locale': 'ja',
+      'replaceSync': {
+        'Removing combatant Alexander Prime': 'Removing combatant Alexander Prime', // FIXME
+        'casts Holy Bleed': 'casts Holy Bleed', // FIXME
+        'casts Void of Repentance': 'casts Void of Repentance', // FIXME
+        'readies Chronofoil': 'readies Chronofoil', // FIXME
```

Comment:
Oof.  At some point before the next expansion, I was going to add better alex normal and savage timelines and triggers.  These are all really old and not made by me.  I think I would almost rather just leave all of Alex savage alone for now and fix them up once they're better.

Reply:
So are you fine if i just remove all alex from this PR?

Reply:
Yeah, I think removing all of alex is the best at this point.  Sorry for the churn.  I should have said something earlier.

Reply:
ok i removed them for now. Will open an issue so we can keep track.